assets/src/blocks/Media/MediaEditor.js

```diff
@@ -0,0 +1,155 @@
+
+  return (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+          <URLInput
+            label={__('Media URL/ID', 'planet4-blocks-backend')}
+            placeholder={__('Enter URL', 'planet4-blocks-backend')}
+            value={ mediaURL }
+            onChange={
+              value => {
+                setMediaURL(value)
+                debouncedMediaURLUpdate(value)
+              }
+            }
+            help={__('Can be a YouTube, Vimeo or Soundcloud URL or an mp4, mp3 or wav file URL.', 'planet4-blocks-backend')}
+          />
+        </PanelBody>
+      </InspectorControls>
+    </Fragment>
+  );
+}
+
+const renderView = (props, toAttribute) => {
+  const { attributes, setAttributes } = props;
+  const { video_title, description, embedHtml, youtube_id, posterURL } = attributes;
+
+  function onSelectImage({id}) {
+    setAttributes({video_poster_img: id});
+  }
+
+  function onSelectURL({url}) {
+    setAttributes({id: null});
+  }
+
+  function onUploadError({message}) {
+    console.log(message);
+  }
+
+  return (
+    <Fragment>
+      <Tooltip text={__('Edit text', 'planet4-blocks-backend')}>
+        <header className="articles-title-container">
+          <RichText
+            tagName="h2"
+            className="page-section-header"
+            placeholder={__('Enter title', 'planet4-blocks-backend')}
+            value={video_title}
+            onChange={toAttribute('video_title')}
+            keepPlaceholderOnFocus={true}
+            withoutInteractiveFormatting
+            characterLimit={40}
+            multiline="false"
+          />
+        </header>
+      </Tooltip>
+      <RichText
+        tagName="p"
+        className="page-section-description"
+        placeholder={__('Enter description', 'planet4-blocks-backend')}
+        value={description}
+        onChange={toAttribute('description')}
+        keepPlaceholderOnFocus={true}
+        withoutInteractiveFormatting
+        characterLimit={200}
+      />
+      <MediaPlaceholder
+        labels={{ title: __('Video poster image [Optional]', 'p4ge'), instructions: __('Applicable for .mp4 video URLs only.', 'p4ge')}}
+        icon="format-image"
+        onSelect={ onSelectImage }
+        onSelectURL={ onSelectURL }
+        onError={ onUploadError }
+        accept="image/*"
+        allowedTypes={["image"]}
+      />
+      {
+        (youtube_id && !youtube_id.endsWith('.mp4')) && !embedHtml
+        ? <div className="block-edit-mode-warning components-notice is-error">
+            { __( 'The video URL could not be parsed.', 'planet4-blocks' ) }
+          </div>
+        : <Fragment>
+            {
+              youtube_id && youtube_id.endsWith('.mp4')
+              ? <MediaElementVideo videoURL={ youtube_id } videoPoster={ posterURL } />
+              : <MediaEmbedPreview html={ embedHtml ? embedHtml : null } />
```

Comment:
This could be shortened to `embedHtml || null`

Reply:
Can't we even shorten it more to just `embedHtml`?